### PR TITLE
First PR

### PR DESCRIPTION
**Pull request by JonasOstir**
```
let circleCi: string = 'cool';
```
;-)

### CODE REVIEW

File: .scripts/get-latest-chrome.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```
Comment (JonasOstir) [inline .scripts/get-latest-chrome.sh:1]: Nice

File: karma.conf.js [hunk 2]
```diff
@@ -36,7 +36,7 @@ module.exports = function (config) {
               : ['progress', 'kjhtml'],
     port: 9876,
     colors: true,
-    logLevel: config.LOG_INFO,
+    logLevel: config.DISABLED,
```
Comment (JonasOstir) [inline karma.conf.js:39]: Change config to something else\
- [x] and this one too
- [x] check this checkbox

File: .gitignore [hunk 3]
```diff
@@ -40,3 +40,5 @@ testem.log
 # System Files
 .DS_Store
 Thumbs.db
+/node/
```
Comment (JonasOstir-LS) [inline .gitignore:43]: Those are needed for maven right?

File: karma.conf.js (hunk 2 above)
Comment (JonasOstir-LS) [inline karma.conf.js:39]: Change config level:
- [ ] change to something else

**Review by JonasOstir-LS — CHANGES_REQUESTED** [review]

**Review by JonasOstir-LS — APPROVED** [review]

**Pull request merged**